src/configuration.ts

```diff
@@ -1,15 +1,42 @@
 import { AppInfo } from "./appInfo.js"
 import { isString } from "./common/type-checks.js"
-import { appId, Permissions } from "./permissions.js"
+import { appId, Permissions, ROOT_FILESYSTEM_PERMISSIONS } from "./permissions.js"
 
 
 // CONFIGURATION
 
 
 export type Configuration = {
   namespace: string | AppInfo
+
+  /**
+   * Capabilities settings.
+   */
+  capabilities?: {
+    /**
+     * Capability collection settings.
+     */
+    collect?: {
+      /**
+       * If set to `true` then collection won't happen automatically
+       * when creating a `Program` (via `program` or `assemble`).
+       *
+       * @default false
+       */
+      manually?: boolean
```

Comment:
Open to naming suggestions 👀

Reply:
hmm 🤔 how about `manual` or maybe `automatic`?